templates/web/prod/WorkZone/Macros.html.twig

```diff
@@ -668,12 +668,16 @@
 {% macro make_expose_bloc(app, WorkZone) %}
     <div id="expose_workzone" class="expose_workzone">
         <div id="expose_sel" class="custom_select_dark" style="padding:10px;">
-            {% set expose_list= app['conf'].get(['phraseanet-service', 'expose-service', 'exposes']) %}
-
+            {% set expose_list = app['conf'].get(['phraseanet-service', 'expose-service', 'exposes']) %}
+            {% set auth_base_url = app['session'].get('auth_provider.base-url') | trim('/') %}
```

Comment:
get the provider, then it's type, then its options

Reply:
ok